packages/services/src/validateContext.ts

```diff
@@ -233,3 +234,33 @@ export const validateDigitalFacilityRecordContext = (
 
   return { ok: true, value: context };
 };
+
+export const validateAssociationEventContext = (
+  context: IAssociationEventContext,
+): Result<IAssociationEventContext> => {
+  if (_.isEmpty(context.vckit)) return error('Invalid vckit context');
+  if (_.isEmpty(context.epcisAssociationEvent)) return error('Invalid epcisAssociationEvent context');
+  if (_.isEmpty(context.storage)) return error('Invalid storage context');
+  if (_.isEmpty(context.dlr)) return error('Invalid dlr context');
+  if (_.isEmpty(context.identifierKeyPath)) return error('identifierKeyPath not found');
+
+  if (_.isEmpty(context.vckit.vckitAPIUrl)) return error('Invalid vckitAPIUrl');
+  if (_.isEmpty(context.vckit.issuer)) return error('Invalid issuer');
+
+  if (_.isEmpty(context.epcisAssociationEvent.context)) return error('Invalid epcisAssociationEvent context');
+  if (_.isEmpty(context.epcisAssociationEvent.type)) return error('Invalid epcisAssociationEvent type');
+  if (_.isEmpty(context.epcisAssociationEvent.dlrLinkTitle)) return error('Invalid epcisAssociationEvent dlrLinkTitle');
+  if (_.isEmpty(context.epcisAssociationEvent.dlrVerificationPage))
+    return error('Invalid epcisAssociationEvent dlrVerificationPage');
+  if (_.isEmpty(context.epcisAssociationEvent.dlrIdentificationKeyType))
+    return error('Invalid epcisAssociationEvent dlrIdentificationKeyType');
+
```

Comment:
Should we include a check for the `renderTemplate`?

Reply:
I think `renderTemplate` is optional, is it correct?

Reply:
Good catch @huynguyen-hl.